screens/Scan/ScanLayoutController.ts

```diff
@@ -173,9 +173,11 @@ export function useScanLayout() {
       message: t('status.bleError.message', {
         vcLabel: vcLabel.singular,
       }),
-      hint: t('status.bleError.hint', {
-        code: bleError.code,
-      }),
+      hint:
+        bleError.code &&
```

Comment:
Would this work?
Shouldn't we use ternary operator here? Something like below

`controller.bleError.code ? t(...) : ''`

Reply:
Since undefined is a valid value for disabling hint. Ternary operator is not required.